### PR TITLE
Added anishbadhri

### PR DESCRIPTION
**Pull request by anishbadhri**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by elisemoe** [comment]
Hi @anishbadhri 👋😄 we can't wait to celebrate graduation with you! Be sure to submit the form before 12:00 AM PT today (in ~3hr) to be considered for swag.

**Comment by anishbadhri** [comment]
Sorry for the delay, I have updated the shipping form.

File: _data/anishbadhri/anishbadhri.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Anish Badri R S # No longer than 18 characters
+institution: SSN College of Engineering # no longer than 58 characters
+profile_pic: anishbadhri.png # Relative path to the image you uploaded
+quote: "It was a fun 4 years" # No longer than 100 characters
```
Comment (Mohammed-Shoaib) [inline _data/anishbadhri/anishbadhri.md:5]: Remove double quotes from begin and end.

Comment (anishbadhri) [reply _data/anishbadhri/anishbadhri.md]: Done, thanks

**Comment by anishbadhri** [comment]
Hi, I'm still unable to perform the merge even after your approval

**Comment by Mohammed-Shoaib** [comment]
Hey @anishbadhri,

Your pull request is perfect! :confetti_ball: I approved the changes. :+1:

However, I cannot merge it as I do not have write access. You'll have to wait for a reviewer with _write_ access to merge your pull request.

Till then sit back, relax, and enjoy :relaxed: 

Best,
Mohammed Shoaib

**Pull request merged**